wrappers/javascript/packages/anoncreds-react-native/src/ReactNativeAnoncreds.ts

```diff
@@ -392,4 +392,192 @@ export class ReactNativeAnoncreds implements Anoncreds {
     const handle = this.handleError(this.anoncreds.keyCorrectnessProofFromJson(serializeArguments(options)))
     return new ObjectHandle(handle)
   }
+
+  public createW3CCredential(options: {
+    credentialDefinition: ObjectHandle
+    credentialDefinitionPrivate: ObjectHandle
+    credentialOffer: ObjectHandle
+    credentialRequest: ObjectHandle
+    attributeRawValues: Record<string, string>
+    revocationConfiguration?: NativeCredentialRevocationConfig
+    encoding?: string
+  }): ObjectHandle {
+    const attributeNames = Object.keys(options.attributeRawValues)
+    const attributeRawValues = Object.values(options.attributeRawValues)
+
+    const credential = this.handleError(
+      // eslint-disable-next-line @typescript-eslint/ban-ts-comment, @typescript-eslint/prefer-ts-expect-error
+      // @ts-ignore
+      this.anoncreds.createW3CCredential({
+        // eslint-disable-next-line @typescript-eslint/ban-ts-comment, @typescript-eslint/prefer-ts-expect-error
+        // @ts-ignore
+        ...serializeArguments(options),
```

Comment:
Would it be possible to fix these @ts-ignores? 

It will disable any type checking for this line, and can result in type issues down the line